### ISSUE TITLE
bug: `functional/prefer-tacit` has false positive in getter

### ISSUE THREAD

**Issue by baseballyama** [eslint-functional/eslint-plugin-functional#964]
## Bug Report

When using a getter, the `functional/prefer-tacit` rule reports an issue, but I believe this is incorrect.

### Expected behavior

No error

### Actual behavior

A lint error occurred.

### Steps to reproduce

```ts
const foo = () => {
  const bar = () => {
    return 1;
  }

  return {
    get baz() {
      return bar();
    },
  };
};
```

### Proposed changes

Skip the rule for getter.
